EN/Moodles_EN.txt

```diff
@@ -1,32 +1,32 @@
 Moodles_EN = {
 	-- Description of moodle:
 	-- Endurance
-	Moodles_endurance_desc_lvl1 = "25%^ Exhausted<br>foraging vision reduced<br>-50% melee damage<br>-0.07 attack speed<br>+10% trip chance while vaulting<br>-5% chance of climbing over tall fences<br>-15% speed",
-	Moodles_endurance_desc_lvl2 = "50%^ Exhausted<br>less foraging vision<br>-80% melee damage<br>-0.14 attack speed<br>+20% trip chance while vaulting<br>-10% chance of climbing over tall fences<br>-30% speed<br>can't sprint",
-	Moodles_endurance_desc_lvl3 = "75%^ Exhausted<br>even less foraging vision<br>-90% melee damage<br>-0.21 attack speed<br>+30% trip chance while vaulting<br>-15% chance of climbing over tall fences<br>-45% speed<br>can't sprint<br>can't run<br>can't speed walk<br>can't exercise",
-	Moodles_endurance_desc_lvl4 = "90%^ Exhausted<br>worse foraging vision<br>-95% melee damage<br>-0.28 attack speed<br>+40% trip chance while vaulting<br>-20% chance of climbing over tall fences<br>-60% speed<br>can't sprint<br>can't run<br>can't speed walk<br>can't exercise<br>can't use heavy weapons",
+	Moodles_endurance_desc_lvl1 = "25%^ Exhausted<br>foraging vision reduced<br>-50% melee damage<br>-7% attack speed<br>+10% trip chance while vaulting<br>-5% chance of climbing over tall fences<br>-19% speed<br>-2% frontal zombie attack blockage",
```

Comment:
Where did you find the "-2% frontal zombie attack blockage"?

Reply:
in IsoGameCharacter.java

var2 defines the luck off blocking the attack

```java
         var2 += this.getWeaponLevel() * 3;
         var2 += this.getPerkLevel(PerkFactory.Perks.Fitness) * 2;
         var2 += this.getPerkLevel(PerkFactory.Perks.Strength) * 2;
         var2 -= this.getSurroundingAttackingZombies() * 5;
         var2 -= this.getMoodles().getMoodleLevel(MoodleType.Endurance) * 2;
         var2 -= this.getMoodles().getMoodleLevel(MoodleType.HeavyLoad) * 2;
         var2 -= this.getMoodles().getMoodleLevel(MoodleType.Tired) * 3;
```

Which means 3% per level of weapon (+30% on a "knifeDeath" event) + 2% per fitness lvl + 2% per strength lvl - 5% per zombies, -2% for each endurance/heavyLoad -3% foreach tired

Reply:
Oh, nice find.
I'll suggest a rename however to make it easier to understand. 👍